modules/internal/ChapelIteratorSupport.chpl

```diff
@@ -119,8 +105,8 @@ module ChapelIteratorSupport {
     return chpl_buildStandInRTT(instanceObj);
```

Comment:
If we keep "static field type", it's probably worth making this case use "static field type".